eventstore/whatsapp_actions.py

```diff
@@ -74,7 +75,22 @@ def handle_event(event):
 
 
 def handle_whatsapp_delivery_error(event):
-    async_handle_whatsapp_delivery_error.delay(f"whatsapp:{event.recipient_id}")
+    if not event.fallback_channel:
+        async_handle_whatsapp_delivery_error.delay(f"whatsapp:{event.recipient_id}")
+    else:
```

Comment:
This method's name is `handle_whatsapp_delivery_error`, so it doesn't make sense that we're processing SMS/fallback channel events in here. I would make another function, `handle_fallback_delivery_error`, and call it from the `handle_event` method.

Also, this method is only called if `event.is_whatsapp_failed_delivery_event` is `True`, which won't be the case for fallback channel messages, so there's no way to get to this code if we're calling the `handle_event` function.